geonode_mapstore_client/client/js/components/MainEventView/MainEventView.jsx

```diff
@@ -19,7 +18,7 @@ function MainEventView({
             <div className="gn-main-event-content">
                 <div className="gn-main-event-text">
                     <div className="gn-main-icon">
-                        <Icon glyph={icon} />
+                        <Glyphicon glyph={icon} />
```

Comment:
The default icon is exclamation, is it supported by Glyphicon?
Please check all the usage of <MainEventView/> to verify if other icons are used